### PR TITLE
First Solurion in jv-lottery 

### PR DESCRIPTION
**Pull request by banderan**

### CODE REVIEW

File: src/main/java/core/basesyntax/Application.java [hunk 1]
```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.getRandomBall());
+        System.out.println();
+        System.out.println(lottery.getRandomBall());
+        System.out.println();
+        System.out.println(lottery.getRandomBall());
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Application.java:11]: image

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 2]
```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/ColorSupplier.java:7]: image

**Review by Rommelua — CHANGES_REQUESTED** [review]
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery

Before you create PR with your homework, you need to go through the checklist under the task and correct **all the points** described there.

**Comment by banderan** [comment]
Done
I add loop in "Application" class and i cant find any variable method which should be in class level

File: src/main/java/core/basesyntax/Application.java [hunk 3]
```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery[] lottery = {new Lottery(), new Lottery(), new Lottery()};
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Application.java:6]: We do not need 3 Lottery objects.

File: src/main/java/core/basesyntax/Colors.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Colors.java:3]:
```suggestion
public enum Color {
```

File: src/main/java/core/basesyntax/Colors.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    RED,
+    BLUE,
+    PINK,
+    GREEN,
+    BLACK,
+    WHITE
+
+}
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Colors.java:11]:
```suggestion
    WHITE
}
```

File: src/main/java/core/basesyntax/Lottery.java [hunk 6]
```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public String getRandomBall() {
+        int randomInt = random.nextInt(101);
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Lottery.java:10]: image

File: src/main/java/core/basesyntax/Lottery.java [hunk 7]
```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public String getRandomBall() {
```
Comment (Rommelua) [inline src/main/java/core/basesyntax/Lottery.java:9]:
```suggestion
    public Ball getRandomBall() {
```

**Review by Rommelua — CHANGES_REQUESTED** [review]

**Pull request closed without merging**